notes/2020/2020_10_23.md

```diff
@@ -0,0 +1,59 @@
+# PyLadies Global Meeting
+
+- Date: Oct 23 2020
+- Agenda: [Issue #78](https://github.com/pyladies/global-organizing/issues/78)
+- Current meeting attendees: @cecivieira @JUpani @econchick @sonacrystal @Marie-L @valerybriz @reshamas @trallard 
+
+
+## Project Updates
+1. Project Comms (Slack: #project-communication / Repo: https://github.com/pyladies/project-communications)
+None this time
+
+2. Project Admin & Logistics (Slack: #project-admin-logistics / Repo: https://github.com/pyladies/project-admin-logisitics )
+None this time
+
+3. Project Tech & Infra (Slack: #project-tech-infra / Repo: https://github.com/pyladies/pyladies )
+None this time
+
```

Comment:
There is also the CoC team with no updates yet

Reply:
@trallard Do you know how to submit suggestions via GitHub?  If yes, that would be quickest for @cecivieira to accept here.